tests/data/frame_test_list_with_offset.txt

```diff
@@ -0,0 +1,2 @@
+test_imgs 2 5 127
```

Comment:
are you defining a new dataset annotation format?

Reply:
what does each column mean?

Reply:
I add an optional format for ann_file, instead of 'frame_dir num_frame label[s]', it can be 'frame_dir **start_idx** num_frame label[s]'. My code is compatible with original format.

Reply:
what `num_frame` in this context mean? the length of the video or the length of the segment?

Reply:
The length of the video clip (which is part of the entire untrimmed video).

Reply:
I think we need a doc page to describe all possible annotation formats we used. @dreamerlin

Reply:
We can prepare a `annotation_description.md` in `docs/` to describe all possible annotation formats.